### PR TITLE
GUVNOR-3236: Rearrange menus structure

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Comment by paulovmr** [comment]
Part of an ensemble:
* https://github.com/AppFormer/uberfire/pull/798
* https://github.com/kiegroup/kie-wb-common/pull/1053
* https://github.com/kiegroup/drools-wb/pull/572
* https://github.com/kiegroup/kie-wb-distributions/pull/578

File: kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/AdminCustomMenuBuilder.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        this.menuItem = menuItem;
+
+        menuItem.setup("fa fa-cog",
+                       DefaultWorkbenchConstants.INSTANCE.Admin(),
+                       () -> placeManager.goTo(PerspectiveIds.ADMIN));
+    }
+
+    @Override
+    public void push(MenuFactory.CustomMenuBuilder element) {
+        // Do nothing
```
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/AdminCustomMenuBuilder.java]: warning at least?

File: kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/HelpCustomMenuBuilder.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        this.menuItem = menuItem;
+
+        menuItem.setup("fa fa-question",
+                       DefaultWorkbenchConstants.INSTANCE.Help(),
+                       () -> placeManager.goTo(PerspectiveIds.ADMIN));
+    }
+
+    @Override
+    public void push(MenuFactory.CustomMenuBuilder element) {
+        // Do nothing
```
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/HelpCustomMenuBuilder.java]: warning.

File: kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/AdminCustomMenuBuilder.java (hunk 1 above)
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/AdminCustomMenuBuilder.java]: Done.

File: kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/HelpCustomMenuBuilder.java (hunk 2 above)
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/custom/HelpCustomMenuBuilder.java]: Done.

**Pull request merged**